clix/clix.py

```diff
@@ -33,14 +44,16 @@ def OnKeyPress(event):
         active = 0
         prev_Key = None
 
-    elif event.Key == 'c' and prev_Key == 'Control_L':
-        text = xerox.paste(xsel=True)
+    elif event.Key == 'c' or event.Key == 'C' and prev_Key == key_binding[0]:
```

Comment:
This is ofcourse a bug!
@adarsh1021 please check prev_Key against `Ctrl` only. And since its value is different in case of linux and windows, please make changes accordingly.

Thanks @dibyadas